speech/unit_tests/test_client.py

```diff
@@ -192,45 +214,48 @@ def test_sync_recognize_with_empty_results_no_gax(self):
 
     def test_sync_recognize_with_empty_results_gax(self):
         from google.cloud._testing import _Monkey
-        from google.cloud.speech import _gax as MUT
+
         from google.cloud import speech
+        from google.cloud.speech import _gax as MUT
```

Comment:
Nit: `MUT` stands for module-under-test. So in `test_client` the module-under-test is `client`, not `_gax`.